test/units/module_utils/facts/test_date_time.py

```diff
@@ -29,14 +28,14 @@ class FakeNow:
         def fromtimestamp(cls, timestamp):
             return DT
 
-        @classmethod
-        def utcfromtimestamp(cls, timestamp):
-            return DT_UTC
-
     def _time():
         return EPOCH_TS
 
+    def utcfromtimestamp(timestamp):
+        return DT
```

Comment:
Even though the code under test currently doesn't use `tzinfo`, it would be a good idea for the returned value to accurately represent what would be returned:

```suggestion
        return DT.replace(tzinfo=UTC)
```

Reply:
Fixed in `FakeNow.fromtimestamp()`. I removed this function now that datetime.datetime.fromtimestamp is called with Python 3 and 2.